core/chaincode/lifecycle/deployedcc_infoprovider_test.go

```diff
@@ -236,6 +240,114 @@ var _ = Describe("ValidatorCommitter", func() {
 		})
 	})
 
+	Describe("AllChaincodesInfo", func() {
+		var (
+			fakeStateIteratorKVs MapLedgerShim
+		)
+		BeforeEach(func() {
+			fakeStateIteratorKVs = MapLedgerShim(map[string][]byte{})
+			err := resources.Serializer.Serialize("namespaces", "cc-name", &lifecycle.ChaincodeDefinition{}, fakeStateIteratorKVs)
+			Expect(err).NotTo(HaveOccurred())
+			fakeQueryExecutor.GetStateRangeScanIteratorStub = func(namespace, begin, end string) (commonledger.ResultsIterator, error) {
+				fakeResultsIterator := &mock.ResultsIterator{}
+				i := 0
+				for key, value := range fakeStateIteratorKVs {
+					if key >= begin && key < end {
+						fakeResultsIterator.NextReturnsOnCall(i, &queryresult.KV{
+							Key:   key,
+							Value: value,
+						}, nil)
+						i++
+					}
+				}
+				return fakeResultsIterator, nil
+			}
+		})
+
+		It("returns deployed chaincodes info for new lifecycle chaincodes", func() {
+			res, err := vc.AllChaincodesInfo("channel-name", fakeQueryExecutor)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(res).To(HaveLen(1))
+			// because ExplicitCollectionConfigPkg is a protobuf object, we have to compare individual fields
```

Comment:
leftover comment?

Reply:
It is not a leftover comment. It explains why we have to compare individual fields instead of entire object.